tsl/test/sql/continuous_aggs_ddl.sql.in

```diff
@@ -315,3 +315,21 @@ AS SELECT time_bucket('6', time_bucket), SUM(count)
 \set ON_ERROR_STOP 1
 
 DROP INDEX new_name_idx;
+
+CREATE TABLE metrics(time timestamptz, device_id int, v1 float, v2 float);
+SELECT create_hypertable('metrics','time');
+
+-- check expressions in view definition
+CREATE VIEW cagg_expr WITH (timescaledb.continuous)
+AS
+SELECT
+  time_bucket('1d', time) AS time,
+  'Const',
+  first(metrics,time),
+  CASE WHEN true THEN 'foo' ELSE 'bar' END,
+  COALESCE(NULL,'coalesce'),
+  avg(v1) + avg(v2) AS avg1,
+  avg(v1+v2) AS avg2
+FROM metrics
+GROUP BY 1;
+
```

Comment:
I think it would be good to run at least one REFRESH MATERIALIZED VIEW on this view.